packages/react-components/src/components/VideoTile.tsx

```diff
@@ -13,9 +13,9 @@ import {
   iconContainerStyle,
   overlayContainerStyles,
   rootStyles,
-  tileInfoStackItemStyle,
```

Comment:
Can this style be deleted now? Or is it used elswhere?

Reply:
> > Is the larger overlay size intended?
> > ![image](https://user-images.githubusercontent.com/82062616/141009493-7bf89af4-eecc-45d3-80a4-f9abaa206ac8.png)
> 
> Err -> smaller (it's larger before this PR)

Will try to maintain the old padding. Thanks.

Reply:
Yes . This was used for Stack.Item that I removed with style {display: 'flex'} which didn't really do anything.